ticker-sample/src/main/res/layout/activity_main.xml

```diff
@@ -11,26 +10,23 @@
         android:layout_height="wrap_content"
         android:gravity="center"
         android:padding="@dimen/activity_vertical_margin"
-        app:ticker_textColor="@color/colorPrimary"
-        app:ticker_textSize="50sp" />
+        android:textAppearance="@style/TickerTextAppearance" />
 
     <com.robinhood.ticker.TickerView
         android:id="@+id/ticker2"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:gravity="center"
         android:padding="@dimen/activity_vertical_margin"
-        app:ticker_textColor="@color/colorPrimary"
-        app:ticker_textSize="50sp" />
+        android:textAppearance="@style/TickerTextAppearance" />
```

Comment:
Have you tested that adding `android:textSize="100dp"` here overrides the `textAppearance` value?